pandas/core/arrays/categorical.py

```diff
@@ -2126,6 +2126,10 @@ def min(self, skipna=True):
         min : the minimum of this `Categorical`
         """
         self.check_for_ordered("min")
+
+        if len(self._codes) == 0:
+            return np.nan
```

Comment:
This should likely be the right NA value for the dtype. See `na_value_for_dtype`.